packages/indexer/bin/index-files.js

```diff
@@ -1,21 +1,5 @@
 #!/usr/bin/env node
 import process from 'node:process';
-import { execSync } from 'node:child_process';
 import { run } from '../dist/lib/cli.js';
 
-if ((process.env.NODE_OPTIONS ?? '').includes('--no-warnings')) {
-  run(process.argv);
-} else {
-  // Silence experimental warnings
-  try {
-    execSync(process.argv.join(' '), {
-      stdio: 'inherit',
-      env: {
-        ...process.env,
-        NODE_OPTIONS: `${process.env.NODE_OPTIONS ?? ''} --no-warnings`,
-      },
-    });
-  } catch {
-    // Do nothing
-  }
-}
+run(process.argv);
```

Comment:
I am confused. What's the fix here?

Reply:
Removed code was only to silence warnings, but on Windows it introduces a path issue because of exec.